xonsh/environ.py

```diff
@@ -1076,6 +1076,19 @@ class GeneralSetting(Xettings):
         False,
         "Set to ``True`` to show completers invoked and their return values.",
     )
+    XONSH_TRACE_SUBPROC_FUNC = Var.with_default(
+        None,
+        doc=(
+            "A callback function used to format the trace output shown when $XONSH_TRACE_SUBPROC=True."
+        ),
+        doc_default="""\
+By default it just prints ``cmds`` like below.
+
+.. code-block:: python
+def tracer(cmds: list, captured: Union[bool, str]):
+    print(f"TRACE SUBPROC: {cmds}, captured={captured}", file=sys.stderr)
+""",
+    )
```

Comment:
Style nit -- since all the other `doc_defaults` here use implicit string concatenation, can we do the same here?

Reply:
I want to keep this way. It is cleaner and easier to edit. I remember breaking some long lines in this file that resulted in the  implicit string concats here, I didn't wanted to mess up the whitespaces.

Reply:
Ok, that's fair.  Let's try, as changes are made to the descriptions (if they are) to push folks to also convert to the triple-quoted blocks.